person.rb

```diff
@@ -1,68 +1,38 @@
-class Namable
-  def correct_name
-    raise NotImplementedError('Nameable has not implemented correct_name method')
-  end
-end
+require './validate'
+require './rental'
 
-class Person < Namable
+class Person
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] `:rentals` should be part of the `attr_accessor`. It will be nice to fix this.